app/controllers/applications_controller.rb

```diff
@@ -1,14 +1,39 @@
 class ApplicationsController < ApplicationController
-  before_action :require_admin, only: [:admin_destroy]
```

Comment:
Removing the two `before_action`s here looks really good to me. It visible increases the focus and responsibility of this controller.

Reply:
🌻